### ISSUE TITLE
bug: React/Ionic ion-datetime scroller lags 

### ISSUE THREAD

**Issue by maylorsan** [ionic-team/ionic-framework#26532]
### Prerequisites

- [X] I have read the [Contributing Guidelines](https://github.com/ionic-team/ionic-framework/blob/main/.github/CONTRIBUTING.md#creating-an-issue).
- [X] I agree to follow the [Code of Conduct](https://ionicframework.com/code-of-conduct).
- [X] I have searched for [existing issues](https://github.com/ionic-team/ionic-framework/issues) that already report this problem, without success.

### Ionic Framework Version

- [ ] v4.x
- [ ] v5.x
- [X] v6.x
- [ ] Nightly

### Current Behavior

We using ion-datetime with min restriction: `min={new Date().toISOString()}` 
When you press the button to scroll through the month, the scroller jumps and lags.

When I delete `min={new Date().toISOString()}` props everything is fixed

Here is code:

`
import {IonContent, IonDatetime, IonHeader, IonPage, IonTitle, IonToolbar} from '@ionic/react';
import ExploreContainer from '../components/ExploreContainer';
import './Tab3.css';

const Tab3: React.FC = () => {
  return (
    <IonPage>
      <IonHeader>
        <IonToolbar>
          <IonTitle>Tab 3</IonTitle>
        </IonToolbar>
      </IonHeader>
      <IonContent fullscreen>
        <IonHeader collapse="condense">
          <IonToolbar>
            <IonTitle size="large">Tab 3</IonTitle>
          </IonToolbar>
        </IonHeader>
          <IonDatetime
              locale={'uk'}
              presentation="date"
              multiple={true}
              min={new Date().toISOString()}
              firstDayOfWeek={1}
          />
        <ExploreContainer name="Tab 3 page" />
      </IonContent>
    </IonPage>
  );
};

export default Tab3;
`
https://user-images.githubusercontent.com/51034456/209315118-1e34201f-95e4-4c05-937c-1584ac3bd20c.mov

### Expected Behavior

When I click on the scroller I should be able to scroll and find the correct month/year

### Steps to Reproduce

Create project via `ionic start myApp tabs --capacitor`
Insert next code: 
`<IonDatetime presentation="date" min={new Date().toISOString()} />`

### Code Reproduction URL

https://github.com/maylorsan/datetime

### Ionic Info

```
Ionic:

   Ionic CLI       : 6.20.3 (/usr/local/lib/node_modules/@ionic/cli)
   Ionic Framework : @ionic/react 6.4.1

Capacitor:

   Capacitor CLI      : 4.6.1
   @capacitor/android : not installed
   @capacitor/core    : 4.6.1
   @capacitor/ios     : not installed

Utility:

   cordova-res : not installed globally
   native-run  : 1.7.1

System:

   NodeJS : v16.18.0 (/usr/local/bin/node)
   npm    : 8.19.2
   OS     : macOS
```

**Comment by amandaejohnston** [comment]
Thank you for the issue. I'm unable to replicate the lag behavior using the linked repo; could you give more details on what browser and device you are testing on?

**Comment by maylorsan** [comment]
Yes, now it works well, I assume that the problem was in the last month of the year

**Comment by amandaejohnston** [comment]
After setting my computer's date to Dec 23, 2022, I can replicate the issue where the selected month keeps jumping back to December, but that is a duplicate of https://github.com/ionic-team/ionic-framework/issues/25293, so I'll set that aside for now.

I'm still unable to replicate the lag when scrolling, though. If it still replicates for you, could you give some more details?
- What browser and device are you testing on?
- What locale code are you using? (Examples: `en-US`, `es-MX`, `ru-RU`)
- Any other info you can think of that might help me replicate the behavior?

**Comment by maylorsan** [comment]
**MasOS:** 13.0.1 (arm64)
**Chrome:** Version 108.0.5359.98(arm64)

**Locale used in issue**: uk-UK

as i say before this lags are happening only when `IonDatetime` component have `min` props.

Uploading Screen Recording 2022-12-22 at 08.12.31.mov…

**Comment by maylorsan** [comment]
no, for now i can't reproduce this lag
I will close this issue as duplicate of https://github.com/ionic-team/ionic-framework/issues/25293

@amandaejohnston thank you)
